.vscode/settings.json

```diff
@@ -4,6 +4,14 @@
 	},
 	"editor.defaultFormatter": "esbenp.prettier-vscode",
 	"editor.formatOnSave": true,
-	"eslint.rules.customizations": [{ "rule": "*", "severity": "warn" }],
-	"typescript.tsdk": "node_modules/typescript/lib"
+	"eslint.rules.customizations": [
+		{
+			"rule": "*",
+			"severity": "warn"
+		}
+	],
+	"typescript.tsdk": "node_modules/typescript/lib",
+	"[jsonc]": {
+		"editor.defaultFormatter": "esbenp.prettier-vscode"
+	}
```

Comment:
[Request] Can you revert these unrelated changes please? If you want to add them in we can discuss in a new issue.